pkg/kubelet/prober/prober_test.go

```diff
@@ -333,3 +333,40 @@ func TestNewExecInContainer(t *testing.T) {
 		}
 	}
 }
+
+func TestNewProber(t *testing.T) {
+	runner := &containertest.FakeContainerCommandRunner{}
+	recorder := &record.FakeRecorder{}
+	prober := newProber(runner, recorder)
+
+	if prober == nil {
+		t.Error("Expected a non-nil prober")
+	}
+	// Add more assertions as needed to validate the initialization of prober
```

Comment:
@jralmaraz Please, consider testing events. Please, use previous PRs, e.g. https://github.com/kubernetes/kubernetes/pull/116030/files#diff-a4e1cdbcf3b09c95dcabb07e531db97347f01c1d5abc151d867e9221a1caa47cR338-R409

Reply:
I've reviewed the other PRs and added testing of events as well, let me know if it look ok! Thanks a lot!